frontend/src/components/team/Card.jsx

```diff
@@ -1,25 +1,28 @@
 import React from 'react';
 import { useHistory } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-import { CardSection, StyledSpan } from './Card.styles';
-import { H4 } from './SelectedTeam.styles';
+import { CardSection, StyledSpan, CardHeading } from './Card.styles';
+//import {Link} from 'react-router-dom';
 
 const Card = ({ team, numberOfSeries }) => {
     const [t] = useTranslation(['team']);
     let history = useHistory();
-
     return (
-        <CardSection
+        <div
```

Comment:
What's the reason for `div` instead of `a`? I think it's best practise to use semantic HTML always when possible.